platform/windows/detect.py

```diff
@@ -169,7 +169,7 @@ def get_flags():
 
 	return [
 		('freetype','builtin'), #use builtin freetype
-		('openssl','builtin'), #use builtin openssl
+		#('openssl','builtin'), #use builtin openssl
```

Comment:
And you're already pretty good with git, your learn fast. I was like that for at least one year before I really started learning stuff:
![](https://imgs.xkcd.com/comics/git.png)
:-P